src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,56 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users", schema = "cinema")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
+    private String hashedPassword;
```

Comment:
```suggestion
    private String password;
```
rename getter and setter as well